MAINTAINERS.md

```diff
@@ -6,6 +6,7 @@ Maintainers are approvers who have shown good technical judgement in guiding fea
 | ------------------------ | ------------------------------------------------------ | -------------------------    | -----------                               |
 | Ettore Di Giacinto       | [@mudler](https://github.com/mudler )                  |  SpectroCloud                | Original Author, Administrator, Maintainer|
 | Dimitris Karakasilis     | [@jimmykarily](https://github.com/jimmykarily )        |  SpectroCloud                | Administrator, Maintainer                 |
+| Itxaka Serrano Garcia    | [@itxaka](https://github.com/itxaka )                  |  SpectroCloud                | Administrator, Maintainer                 |
```

Comment:
@Itxaka I don't have the character in my keyboard layout, so I hope you will fix it here =D